### ISSUE TITLE
Online community editor has server error generating results 

### ISSUE THREAD

**Issue by hekl** [RMLio/rmleditor-ce#5]
### Description
I am defining an entity in the editor, but when I want to see the results (Run mapping) is gives a server error. See screenshot.
![Screenshot 2021-09-09 at 12-53-40 https app rml io](https://user-images.githubusercontent.com/6750271/132674080-5f99b6c1-56a3-4f20-b810-54e54ebeb7d0.png)

### Steps
Just try to define an entity and run mapping.

### Environment
I am using Firefox 91.0.2 and Ubuntu 20.04

**Comment by pheyvaer** [comment]
Hi @hekl 

Thank you for the issue! What is the input data that you are using?

**Comment by hekl** [comment]
A csv file. I add it here. Taking the url column as its subject entity.
[opleidingen_simple.csv](https://github.com/RMLio/rmleditor-ce/files/7143760/opleidingen_simple.csv)

**Comment by pheyvaer** [comment]
Thanks! I think the issue that you didn't connect the resource (the grey circle) with a data source (the CSV file). If a resource is related to a data source then it will have a different color. As shown in [this video](https://youtu.be/2y3M8QuGZpY?t=201).

**Comment by hekl** [comment]
I have tried in Firefox and Chrome, but I am not able to click the source file radio button.
![image](https://user-images.githubusercontent.com/6750271/132867548-1942dcc9-f73f-48c5-8160-401814bb9943.png)

**Comment by pheyvaer** [comment]
Did you try it using drag and drop, like shown in the video?

**Comment by hekl** [comment]
Yes, I did. From your video I see that when loading the csv file the radio button turns orange. In my case it doesn't. The dragged entity like wise stays grey.

**Comment by pheyvaer** [comment]
But in your case it's blue. There is not difference in meaning besides indicating the different data sources.
